File: BTS1/Programmes/Programmes python/Saisie-bornée-dun-nombre.py
#
#Ghouaiel Elyes
#05/10/2021
#
#saisie bornée d'un nombre
#
#le programme doit demander à l'utilisateur un nombre compris entre 1  et 10 (inclus) jusqu'à que la réponse convienne.
#Début

x=float(input("Saisir un nombre entre 1 et 10 inclus"))
if  1 < x < 10 :
    print("le nombre compris est bien entre 1 et 10 inclus")
else:
    while (x<1 and x>10):
        x=float(input("le nombre n'est pas compris entre 1 et 10 inclus, Reinsérer un nombre"))
#Fin
        

File: BTS1/Programmes/Programmes C++/Exercices-promgrammes-c/Saisie-bornée-dun-nombre/Saisie-bornée-dun-nombre/Saisie-bornée-dun-nombre.cpp
// Saisie bornée d'un nombre.cpp : Ce fichier contient la fonction 'main'. L'exécution du programme commence et se termine à cet endroit.
//
//chenal killian 
//28/09/2021
//
//saisie bornée d'un nombre
//
//le programme doit demander à l'utilisateur un nombre compris entre 1  et 10 (inclus) jusqu'à que la réponse convienne.
//

#include <iostream>
using namespace std;
//déclaration des variables 
float x;

int main() // début
{
	cout << "saisir un nombre compris entre 1 et 10 inclus";
	cin >> x;
	if (x >= 1 && x <= 10)
	{
		cout << "le nombre compris est bien entre 1 et 10 inclus";

	}
	else
	{
		do
		{
			cout << "le nombre n'est pas compris entre 1 et 10";
			cin >> x;
			 
		} while ((x >= 1 && x <= 10);

	}
	
}


